pkg/oc/cli/admin/policy/modify_roles.go

```diff
@@ -593,6 +606,9 @@ func (o *RoleModificationOptions) AddRole() error {
 				}
 			case rbacv1.GroupKind:
```

Comment:
Because we use the RBAC kinds now there is no distinction between the group types.